packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -105,16 +105,16 @@ The `value` object is where you specify the specific operation to perform. It is
 | Property | Description                                                                                                                                                                                                       | Data Type       |
 |----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
 | op       | The name of one of the [supported ops](#list-of-supported-ops).                                                                                                                                                   | String          |
-| path     | Location within the token to apply the operation, specified as a slash-delimited path. When adding a claim, this will always begin with `/claims/`,  and be followed by the name of the new claim you are adding. | String          |
+| path     | Location within the token to apply the operation, specified as a slash-delimited path. When adding, replacing, or removing a claim, this always begins with `/claims/`  and is followed by the name of the new claim that you are adding. When replacing a token lifetime, the path always begins with `/token/lifetime/expiration`. | String          |
```

Comment:
Regarding `When replacing a token lifetime, the path always begins with /token/lifetime/expiration` - I would say `the path should always be equal to ...` not `begins with`